Sources/SwiftGRPCNIO/ClientCalls/BaseClientCall.swift

```diff
@@ -83,6 +83,10 @@ extension BaseClientCall: ClientCall {
     return self.clientChannelHandler.statusPromise.futureResult
   }
 
+  public var trailingMetadata: EventLoopFuture<HTTPHeaders> {
```

Comment:
What's the motivation behind this move? (Not objecting to it, just curious.)

Reply:
It's a workaround for: https://bugs.swift.org/browse/SR-10128

I'll add a comment.